### ISSUE TITLE
Episode 2 Catalyst Updating

### ISSUE THREAD

**Issue by ulvric** [EpicSevenDB/gamedatabase#185]
Previously it appears things catalyst locations were listed as Normal Chapter # or World Chapter #. That being said, now that we have Cidonia, what format for the new levels? Something like Normal 2-1-1 or Normal Cidonia 1-1.

**Comment by RaphaelDDL** [comment]
Hi ulvric,

Thanks for bringing this up.

Yeah, we'll need to differentiate, even more now that World actually doesn't exist and was moved to the side story as well. 

regarding it, it's called side story is called Unrecorded History, and is the Episode 1 Epilogue. That's ok. Problem is, I just tried to zoom out the map on the side story and wasn't able, so maybe it'll be Ritania specific, and we'll have a Cidonia specific later? 

At this point, it'll be speculation but we'll need to assume something, and I'll guess would be a separate side story. So maybe we need to rename all "World" into "Unrecorded History"?

I guess for normal levels, we could do like you suggested, having the continent and then the stage. Like, the ones currently as "Normal 2-1" would become "Ritania 2-1", and new spots @ Cidonia would be "Cidonia 1-1".

@kmalone86 @Dimbreath What you guys think? I think sounds fine being like that, and we need to update YufineBot too so maybe it's a good time to fix on db so you can use the api instead of keeping track on the bot ✌️ 

tl;dr - to do now:
- Rename all "Normal" to "Ritania";
- Rename all "World" to "Unrecorded History";
- Fix any catalysts that were moved/removed when World was removed;
- Add new Cidonia information;

First couple (renames) I'll do today, if you are all ok with it, while couple last we'll build together.

**Comment by RaphaelDDL** [comment]
so I discussed with the guys, and we'll do something a bit different:

```
{
    "name": "Ancient Creature Nucleus",
    "type": "catalyst",
    "rarity": 5,
    "description": "A nucleus with an unknown source of power, some say it can be used to move anything in the world.",
    "apShops": [
        {
            "map": "Ritania",
            "difficulty": "World (Unrecorded History)",
            "chapter": "7 North Wetheric Moor",
            "quantity": 2,
            "cost": 400
        }
    ],
    "locations": [
        {
            "map": "Ritania",
            "difficulty": "World (Unrecorded History)",
            "name": "A Crack in the World",
            "node": "3S-Crack",
            "mobcount": "?"
        },
        {
            "map": "Ritania",
            "difficulty": "World (Unrecorded History)",
            "name": "A Crack in the World",
            "node": "5S-Crack",
            "mobcount": "?"
        },
        {
            "map": "Ritania",
            "difficulty": "World (Unrecorded History)",
            "name": "?",
            "node": "7-4",
            "mobcount": "?"
        },
        {
            "map": "Ritania",
            "difficulty": "World (Unrecorded History)",
            "name": "Seven Flags Fortress",
            "node": "7-S6",
            "mobcount": "?"
        }
    ]
}
```

Map and difficulty will be added, and name and chapter will be changed a bit. I started branch `LOCATIONS`, will take a while but I'll do all 'em

**Comment by Dimbreath** [comment]
[#d05dc9b.](https://github.com/EpicSevenDB/gamedatabase/commit/928fa40889c2e11bbbc50524af1a57678d3dfa58) Note that this is a WIP so it might not be perfect.

For issues on that data will need to open new issue.
